src/lib.rs

```diff
@@ -163,12 +164,19 @@ impl Client {
     /// describes the metadata and data of the key.
     ///
     /// Otherwise, [`Error`] is returned.
-    /// This will eventually be deprecated in favor of `get_multi`
-    pub async fn get_many<I, K>(&mut self, keys: I) -> Result<Vec<Value>, Error>
+    pub async fn get_multi<'a, K>(
+        &mut self,
+        keys: &'a [K],
```

Comment:
If we change this to a `&[u8]` does it remove the need to clone the key?

Reply:
Nick and I went through a quick exercise trying to get this suggestion to work and it seems like we always require either multiple iterations or a secondary hashmap that contains the keys, which is almost certainly more expensive than just cloning the key as we insert into the results hashmap.  The `Vec<u8>` approach feels like the best compromise.